helm/memcached-operator/helm-charts/memcached/README.md

```diff
@@ -48,6 +48,8 @@ The following table lists the configurable parameters of the Memcached chart and
 | `memcached.maxItemMemory`  | Max memory for items (in MB)    | `64`                                                    |
 | `memcached.extraArgs`      | Additional memcached arguments  | `[]`                                                    |
 | `metrics.enabled`          | Expose metrics in prometheus format | false                                               |
+| `metrics.serviceMonitor.enabled`          | Expose serviceMonitor to be scraped in prometheus-operator target | false                                               |
+| `metrics.serviceMonitor.interval`          | Default frequency to scrap metrics | 15s                                               |
 | `metrics.image`            | The image to pull and run for the metrics exporter | A recent official memcached tag      |
```

Comment:
Since we have the above. I think we should add the "helm/memcached-operator/helm-charts/memcached/templates/servicemonitor.yaml" which was added in the chart template as well.